build.sc

```diff
@@ -43,7 +43,7 @@ class arithmetic extends ScalaModule with ScalafmtModule with PublishModule { m
   object tests extends Tests {
     override def ivyDeps = Agg(ivy"com.lihaoyi::utest:latest.integration")
 
-    def testFramework = "utest.runner.Framework"
+    def testFrameworks = Seq("utest.runner.Framework")
   }
```

Comment:
Deprecated in https://github.com/com-lihaoyi/mill#0-9-7-2021-05-14
> TestModule: Added new testFramework target and only support one test framework. Deprecated testFrameworks targets.

Should revert here.